### ISSUE TITLE
coverage with tox

### ISSUE THREAD

**Issue by beauxq** [mCodingLLC/SlapThatLikeButton-TestingStarterProject#2]
I followed your youtube video as a guide in [this repository](https://github.com/beauxq/be8bbbce) and I put this in my pyproject.toml:
```
[tool.pytest.ini_options]
addopts = "--cov=src/be8bbbce/components --cov-branch"
testpaths = [
    "tests",
]
```
When I run `pytest`, I see my statement and branch coverage the way I want, but when I run `tox`, it says I have 0% statement coverage and doesn't show any branch coverage.

How do I get `tox` to show the same coverage as `pytest`?

**Comment by mapyr** [comment]
Hi @beauxq 

I add in `tox.ini` more commands, like:

```ini
commands =
    pytest --basetemp={envtmpdir} --cov --cov-append --cov-report=term-missing
```

And results looks like:

```text
---------- coverage: platform darwin, python 3.9.7-final-0 -----------
Name                                    Stmts   Miss  Cover   Missing
---------------------------------------------------------------------
src/slapping/__init__.py                    0      0   100%
src/slapping/slap_that_like_button.py      20      0   100%
tests/__init__.py                           0      0   100%
tests/conftest.py                          16      1    94%   22
tests/test_slapping.py                     26      3    88%   29, 44-45
---------------------------------------------------------------------
TOTAL                                      62      4    94%
```

**Comment by mapyr** [comment]
Or is possibility to made another step, dedicated to coverage, like:

```ini

[testenv:report]
deps = coverage
skip_install = true
commands =
    coverage report
    coverage html
```

**Comment by mCodingLLC** [comment]
Thanks @pyrkamarcin I think either one of those solutions works great depending on what you want. @beauxq I think that should solve your issue so I'll go ahead and close this.

**Comment by beauxq** [comment]
> ```ini
> commands =
>     pytest --basetemp={envtmpdir} --cov --cov-append --cov-report=term-missing
> ```

If I use this, it shows coverage on my `tests` directory, but nothing from my `src` directory.

**Comment by beauxq** [comment]
I figured it out when I remembered [7:59](https://youtu.be/DhUpxWjOhME?t=479) in the youtube video.

Because I was trying to get coverage on only 1 submodule of my package, I was giving it a directory of where to look: `--cov=src/be8bbbce/components`

But it's not running code in that directory. It's running a link in ...`lib/python/site-packages`

I think I tried `--cov=be8bbbce/components` at some point and found it didn't work, but after I remembered it's running the installed package, I tried `--cov=be8bbbce.components` and it worked.
